src/main/java/io/codiga/plugins/jetbrains/utils/CodePositionUtils.java

```diff
@@ -38,6 +38,16 @@ public static String indentOtherLines(String code, int indentation) {
         return String.join(LINE_SEPARATOR, codeArrayUpdated);
     }
 
+    /**
+     * Know if the current line has tabs indentation
+     *
+     * @param lineOfCode - the line of code.
+     * @return - boolean response if it is
+     */
+    public static boolean detectIfTabs(String lineOfCode) {
+      return lineOfCode.charAt(0) == CHARACTER_TAB;
```

Comment:
what if there is no character available in the line? `lineOfCode.charAt` will throw an exception.